ReSharper.FSharp/test/data/features/daemon/pipeChainTypeHints/Skipped 05 - Inner pipe.fs.gold

```diff
@@ -0,0 +1,99 @@
+﻿if true then
+      1||(0)
+      |> id
+else 1||(1)
+     |> id||(2)
```

Comment:
Is it expected that one branch shows the type and the other does not? Is it to skip the duplicated type?

Reply:
actually, after 
```F#
else 1
     |> id
``` 
hint is shown for the whole if/else expression as a left operand of outer pipe. For the `else` branch, we are indeed not displaying the duplicated type (even if it may be slightly different). For the `then` branch and in the previous branches in the `match`, I have decided not to display the type, as we will show the type for the entire if/match below.

In C#, Linq similarly shows the type for inner Linq only on the last branch for switch expression
![image](https://github.com/JetBrains/resharper-fsharp/assets/26364714/61361a77-d659-49a0-80c3-6a7bdbab6240)